pygeo/constraints/areaConstraint.py

```diff
@@ -421,25 +422,6 @@ def evalFunctionsSens(self, funcsSens, config):
 
             funcsSens[self.name] = tmpTotal
 
-    def _computeArea(self, p0, p1, p2):
```

Comment:
That is correct, but they were doing different things and the name was very vague. I didn't move the other one because it relied on data from the class itself (i.e. it was using `self`).